### ISSUE TITLE
Understanding the code and model better

### ISSUE THREAD

**Issue by AndreyShternshis** [Sajad-Sharhani/Computational-Science-Thesis#8]

**Comment by AndreyShternshis** [comment]
- [ ] add an additional layers so that you can distinguish between latent spaces and two components of the total decision
- [ ] fix alpha = 1, beta =1, plot total loss and the three losses against epochs
- [ ] divide data by training, validation and testing set and plot the training and validation losses
- [ ] run the baseline model that predicts decision from the dataset based on both sensitive and permissible attributes. You can keep the  same structure of neural network as in the desired one but the loss is just cross entropy between predictions and decisions. Calculate fair metric on training and testing sets
- [ ] Plot the 3 losses as a function of beta: conservatism loss (you can call it constancy loss by the way), accuracy loss, fairness loss. Also plot the mean value of predictions to see what is going on.

Let me know if you have any difficulties!
